Makefile

```diff
@@ -44,7 +44,7 @@ endif
 IMAGE_PLATFORMS ?= linux/amd64,linux/arm64
 
 # Base build image to use.
-BUILD_BASE_IMAGE ?= golang:1.22.12@sha256:1f298b0c9fecdf504389a0329236f948cc04a566a2bb32337207cbaaa2f8177c
+BUILD_BASE_IMAGE ?= golang:1.24.2@sha256:86d1854c5e3d0539ccc5eb052ebb60c8d40abf34e3a17833ccb2167931527d22
```

Comment:
Looks like this is the linux/amd64 SHA, can we switch this to the multi-arch manifest for native use on other architectures?

`golang:1.24.2@sha256:1ecc479bc712a6bdb56df3e346e33edcc141f469f82840bab9f4bc2bc41bf91d `

Reply:
Thanks @skriss, good catch! I've fixed it.

By the way, have you been doing these bumps manually or do you have some automation? I was writing some scripts for golang, envoy and golangci-lint version bumps myself.

Reply:
I've done them manually, but agree some scripts would be nice to have, it was always in the back of my mind but never quite got to it.